src/cron/personalizedDigest.ts

```diff
@@ -17,7 +17,7 @@ const cron: Cron = {
     let digestCount = 0;
 
     const personalizedDigestStream = await personalizedDigestQuery.stream();
-    const notifyQueueConcurrency = 1000;
+    const notifyQueueConcurrency = 10;
```

Comment:
Can you turn that into an env var so we will not have to rebuild the image as we try? It'll be enough to just change the variable in pulumi

Reply:
@idoshamun In .infra/Pulumi.prod.yaml?

Reply:
yes

Reply:
It is set from pulumi anyway so it doesn't require to rebuild the image, it's fine :)